lib/inspector.coffee

```diff
@@ -50,7 +56,7 @@ class Inspector
             cursor_pos = cursor.getBufferColumn()
         return [code, cursor_pos]
 
-    getInspector: ->
+    initializeInspector: ->
```

Comment:
I see `initializeInspector` is now used to toggle the inspector view.
I'd still think we could rename `@inspector` to `@view` and move its creation to the constructor.
I'd also rename `@initializeInspector` to `@show`.